# Translational Gloo (project omics_to_omop)

## Overview:

This project will describe how to pass a clinical report format to omop and import data from omop to that report.

## Schedule:

All EST unless otherwise specified

### 6/1

245pm-345pm: Making README - figuring out roles and who is doing what. For next meeting: talk to each time on what they'll generate, talk to Ames about what they'll want to do and data they'll need. 


## Thoughts

This project is effectively acting as the glue between data generated by the other hackathon teams. We are connecting sequencing and molecular data to the patients from which they are generated. Clinical reporting of 'omics!

## Team roles and responsibilities:

- Everyone needs to be a laison between this team and all the other teams 2-5.
- Kamileh: SVs/EVs; Meghana: TCR; Sal: PRS; Nick: team 6 (UX)
- We are asking for 1 table from each team that has the data they are generating and to be reported. We need to know the fields, field types, and possible attributes.
- We then need to have these tables relatable via OMOP via via ROMOPOmics. Nick can do this since he has used ROMOPOmics before. But others should try using it!
- We then want to give the OMOP formatted tables or standardized tables to the UX team 6.

## Our team's ongoing slides on our work

https://docs.google.com/presentation/d/1aH4bp_nyztGk8JRa5dAI_K9LZqQflSXLbefREsRVE0c/edit#slide=id.p

## What other teams should do

- Talk to us!!!
- We need to interoperate and our team can help do that. Then UX should have less headaches. 

## What we currently know from each team
**Chat with Pooja from Team 2 (PRS)**
- Compute PRS from p-value thresh and GWAS (synthetic from UK BioBank) with synthetic phenotypes\
- Input - VCF/genotype (blink file) with expected outcome/phenotype
- Output - PRS, visualization, and predictive model
- Potential Gene level PRS score
  - Looking for geneâ€™s contribution to the score, effect of adding and removing
- The model is the relationship between the PRS score and the phenotype
  - They are gonna work to figure out what it means (level of associatedness/likeihood?)
- We still need the file type, type of outcome

**Chat with Ahmad from Team 6 (UX)**
-For handling the inputs from the various teams, we should be utilizing [snpReportR](https://github.com/collaborativebioinformatics/snpReportR)
  - It will be a reliable skeleton to expand upon to process
  - [Manuscript](https://docs.google.com/document/d/1UMY3YFjeoGHiFWqCg_AZAPXrnZbS4Z52qOeCrvCXWD0/) associated with it for understanding
  - Standalone tool according to 
-SV Team output
  - Annotated VCF file 
  - [GeneVar](https://jmonlong.shinyapps.io/GeneVar/) is a SV browser
  - [GitHub](https://github.com/collaborativebioinformatics/GeneVar) link from the previous hackathon

## Potential product from this team:

A sqlite database of OMOP tables that connect TCR, PRS, and other data from the hackathon teams to patient-level data. 

## Resources:

### The book for OMOP

The chapters 4-10 are the most important to understand clinical data!

https://ohdsi.github.io/TheBookOfOhdsi/

### ROMOPOmics

This software is a step towards connecting sequencing data into OMOP for clinical reporting! Check this out, download and try the vignette

https://github.com/ngiangre/ROMOPOmics
